frontend/src/pages/ActivityReport/index.js

```diff
@@ -169,7 +211,23 @@ function ActivityReport({
             || report.calculatedStatus === REPORT_STATUSES.NEEDS_ACTION);
 
         updateAdditionalData({ recipients, collaborators, availableApprovers });
-        updateFormData(report);
+
+        let shouldUpdateFromNetwork = true;
+
```

Comment:
I'm not sure if there is more elegant way to handle this without recreating the git language.